README.rst

```diff
@@ -56,6 +56,14 @@ Lastly, override the ``EMAIL_BACKEND`` setting within your Django settings file:
 
    EMAIL_BACKEND = 'django_amazon_ses.backends.boto.EmailBackend'
 
+Optionally, you can set the AWS credentials (falls back to boto3 credentials):
```

Comment:
I tested the scenario where `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are unset, and it appears to in a desirable way (things gracefully fall back to other Boto credential providers). Is that what you meant to capture with `falls back to boto3 credentials`?

Reply:
Yes exactly. I've slightly reworded this based on your feedback in an attempt to make it more clear. Please let me know what you think. I'm happy to tweak the wording if you think it could be more clear